examples/testgputext.c

```diff
@@ -0,0 +1,401 @@
+            SDL_GPU_INDEXELEMENTSIZE_32BIT);
+        SDL_PushGPUVertexUniformData(context->cmd_buf, 0, matrices, sizeof(SDL_Mat4X4) * num_matrices);
+
+        int index_offset = 0;
+        for (TTF_GPUAtlasDrawSequence *seq = draw_sequence; seq != NULL; seq = seq->next) {
+            SDL_BindGPUFragmentSamplers(
+                render_pass, 0,
+                &(SDL_GPUTextureSamplerBinding){
+                    .texture = seq->atlas_texture, .sampler = context->sampler },
+                1);
+
+            SDL_DrawGPUIndexedPrimitives(render_pass, seq->num_indices, 1, index_offset, 0, 0);
+
+            index_offset += seq->num_indices;
+        }
+        SDL_EndGPURenderPass(render_pass);
+    }
+}
+
+void free_context(Context *context)
+{
+    SDL_ReleaseGPUTransferBuffer(context->device, context->transfer_buffer);
+    SDL_ReleaseGPUSampler(context->device, context->sampler);
+    SDL_ReleaseGPUBuffer(context->device, context->vertex_buffer);
+    SDL_ReleaseGPUBuffer(context->device, context->index_buffer);
+    SDL_ReleaseGPUGraphicsPipeline(context->device, context->pipeline);
+    SDL_DestroyGPUDevice(context->device);
+    SDL_DestroyWindow(context->window);
+}
+
+int main(int argc, char *argv[])
+{
+    (void)argc;
+    (void)argv;
+
+    check_error_bool(SDL_Init(SDL_INIT_VIDEO | SDL_INIT_EVENTS));
+
+    bool running = true;
+    Context context = { 0 };
+
+    context.window = check_error_ptr(SDL_CreateWindow("GPU text test", 800, 600, 0));
+
+    context.device = check_error_ptr(SDL_CreateGPUDevice(SUPPORTED_SHADER_FORMATS, true, NULL));
+    check_error_bool(SDL_ClaimWindowForGPUDevice(context.device, context.window));
+
+    SDL_GPUShader *vertex_shader = check_error_ptr(load_shader(context.device, true, 0, 1, 0, 0));
+    SDL_GPUShader *fragment_shader = check_error_ptr(load_shader(context.device, false, 1, 0, 0, 0));
+
+    SDL_GPUGraphicsPipelineCreateInfo pipeline_create_info = {
+        .target_info = {
+            .num_color_targets = 1,
+            .color_target_descriptions = (SDL_GPUColorTargetDescription[]){{
+                .format = SDL_GetGPUSwapchainTextureFormat(context.device, context.window),
+                .blend_state = (SDL_GPUColorTargetBlendState){
+                    .enable_blend = true,
+                    .alpha_blend_op = SDL_GPU_BLENDOP_ADD,
+                    .color_blend_op = SDL_GPU_BLENDOP_ADD,
+                    .color_write_mask = 0xF,
+                    .src_alpha_blendfactor = SDL_GPU_BLENDFACTOR_SRC_ALPHA,
+                    .dst_alpha_blendfactor = SDL_GPU_BLENDFACTOR_DST_ALPHA,
+                    .src_color_blendfactor = SDL_GPU_BLENDFACTOR_SRC_ALPHA,
+                    .dst_color_blendfactor = SDL_GPU_BLENDFACTOR_ONE_MINUS_SRC_ALPHA
+                }
+            }},
+            .has_depth_stencil_target = false,
+            .depth_stencil_format = SDL_GPU_TEXTUREFORMAT_INVALID /* Neex to set this to avoid missing initializer for field error */
```

Comment:
```suggestion
            .depth_stencil_format = SDL_GPU_TEXTUREFORMAT_INVALID /* Need to set this to avoid missing initializer for field error */
```